ts/features/bonus/siciliaVola/types/SvVoucherRequest.ts

```diff
@@ -0,0 +1,101 @@
+export type State = {
+  id: number;
+  name?: string;
+};
+export type Region = {
+  id: number;
+  name?: string;
+};
+export type Province = {
+  id: number;
+  name?: string;
+};
+
+export type Municipality = {
+  id: number;
+  name?: string;
+};
+export type University = {
+  universityName: string;
+  state: State;
+  region: Region;
+  province: Province;
+  municipality: Municipality;
+};
+
+export type Company = {
+  businessName: string;
+  vat: string;
+  state: State;
+  region: Region;
+  province: Province;
+  municipality: Municipality;
+};
+
+export type Hospital = {
+  hospitalName: string;
+  state: State;
+  region: Region;
+  province: Province;
+  municipality: Municipality;
+};
+
+export type DisabledVoucherRequest = {
+  category: "disabled";
+  departureDate: Date;
+  returnDate?: Date;
+};
+
+export type StudentVoucherRequest = {
+  category: "student";
+  university: University;
+  departureDate: Date;
+  returnDate?: Date;
+};
+
+export type WorkerVoucherRequest = {
+  category: "worker";
+  underThresholdIncome: boolean;
+  company: Company;
+  departureDate: Date;
+  returnDate?: Date;
+};
+
+export type SickVoucherRequest = {
+  category: "sick";
+  underThresholdIncome: boolean;
+  hospital: Hospital;
+  departureDate: Date;
+  returnDate?: Date;
+};
+
+type PartialDisabledVoucherRequest = Partial<
+  Omit<DisabledVoucherRequest, "category">
+> &
+  Pick<DisabledVoucherRequest, "category">;
+type PartialStudentVoucherRequest = Partial<
+  Omit<StudentVoucherRequest, "category">
+> &
+  Pick<StudentVoucherRequest, "category">;
+type PartialWorkerVoucherRequest = Partial<
+  Omit<WorkerVoucherRequest, "category">
+> &
+  Pick<WorkerVoucherRequest, "category">;
+type PartialSickVoucherRequest = Partial<Omit<SickVoucherRequest, "category">> &
+  Pick<SickVoucherRequest, "category">;
+
+export type VoucherRequest =
+  | DisabledVoucherRequest
+  | StudentVoucherRequest
+  | WorkerVoucherRequest
+  | SickVoucherRequest;
+
+export type PartialVoucherRequest =
+  | PartialSickVoucherRequest
+  | PartialDisabledVoucherRequest
+  | PartialStudentVoucherRequest
+  | PartialWorkerVoucherRequest;
+
+export type SvBeneficiaryCategory = VoucherRequest["category"];
+
+export type AvailableDestination = ReadonlyArray<string>;
```

Comment:
Nitpicking here: `AvailableDestination` represents a number of destinations. I'd rather use a plural `AvailableDestinations`.

The same may apply to other types like `State`, `Region`, `Province`, `Municipality`.

Reply:
Updated in [0e2e97f](https://github.com/pagopa/io-app/pull/3170/commits/0e2e97fea62dbf7f6a18be1467cf27307424dfe9) and in [f169eea](https://github.com/pagopa/io-app/pull/3170/commits/f169eea1bc8a879610a0de3f92d0b5b50d44f5a4). 

I updated `AvailableDestination` as requested but instead of `State`, `Region`, `Province`, `Municipality` that represent a single value, I updated the `AvailableStateState`, `AvailableRegionState`, `AvailableProvinceState`, `AvailableMunicipalityState` type used to model the list of the values.